docs/src/main/sphinx/release/release-422.md

```diff
@@ -0,0 +1,49 @@
+# Release 422 (12 Jul 2023)
+
+## General
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18005`)
```

Comment:
Did we always use this kind of syntax for CTAS?

Reply:
Yes